okapi-core/src/main/java/org/folio/okapi/util/CorsHelper.java

```diff
@@ -21,6 +20,7 @@
 
   public static final String DELEGATE_CORS = "delegate-CORS";
   public static final String DELEGATE_CORS_MODULE_INSTANCE = "delegate-CORS-module-instance";
+  private static final int ACCESS_CONTROL_MAX_AGE = 7200;
```

Comment:
Is there a specific reason to use 2 hours? BTW Spring default is 30 mins.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
Chrome caps at 2 hours.

Okapi answers CORS OPTIONS pre-fetch requests, it does not pass it along to Spring based FOLIO modules.

For consistency across browsers I choose the maximum value that Chrome allows.